_posts/2017/2017-11-13-hackathon-2017-defails.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Hackathon 2017 Details"
+author: dbarrosop
+description: "Details about the hackathon 2017"
+tags: ["news"]
+---
+
+
+## Intro
+
+It's been a year since our last hackathon so it's time for a second take. This year, in addition to giving you a NAPALM update and present a couple of new projects, we will also have a few talks from various other projects leveraging on napalm and a talk from Gabriele Gergino who was the unofficial winner for the non-existent contest we had in the hackathon we held last year :)
```

Comment:
Gerbino :P

Reply:
Damn it! I was so proud I had gotten all the names right in the agenda without having to check...